dev-python/m2crypto/m2crypto-0.31.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
This copyright header is old. Please commit with repoman to fix it automatically.